packages/node_modules/@webex/internal-plugin-devices/test/integration/spec/devices.js

```diff
@@ -17,24 +17,21 @@ describe('plugin-devices', () => {
     let user;
     let webex;
 
-    before('create users', () => testUsers.create({count: 1})
-      .then(([createdUser]) => {
-        user = createdUser;
-      }));
+    beforeEach('create test users and webex instance',
+      () => testUsers.create({count: 1})
+        .then(([createdUser]) => {
+          user = createdUser;
 
-    beforeEach('create the webex instance', () => {
-      webex = new WebexCore({
-        credentials: user.token
-      });
-
-      devices = webex.internal.devices;
-    });
+          webex = new WebexCore({
+            credentials: user.token
+          });
 
-    // This will be implemented in the same pull request that `unregister()` is
-    // implemented.
-    afterEach('attempt to unregister the device', () => {});
+          devices = webex.internal.devices;
+        }));
 
-    after('remove users', () => testUsers.remove([user]));
+    afterEach('unregister the device and remove test users',
+      () => devices.unregister()
+        .then(() => testUsers.remove([user])));
```

Comment:
There is a bug when using a `before()` clause with creating users, see SPARK-112453